### PR TITLE
People can now see all items when scrolling the search page

### PR DESCRIPTION
**Pull request by haavamoa**
…re keyboard is removed when tapping cancel

### Description of Change

- [SearchPage] Made sure keyboard will disapear when scrolling the result.
- [SearchPage] Keyboard is now removed when people tap the cancel button.
- 
### Todos
- [X] I have tested on an Android device.
- [X] I have tested on an iOS device.
- [ ] I have supported accessibility

### CODE REVIEW

File: src/library/DIPS.Mobile.UI/Components/Searching/SearchPage.cs [hunk 1]
```diff
@@ -85,11 +88,20 @@ public SearchPage()
...
+               
+            SearchBar.Unfocus();
+        }
+
         private void OnUnLoaded(object? sender, EventArgs e)
         {
             Loaded -= OnLoaded;
             Unloaded -= OnUnLoaded;
-            SearchBar.Unfocus();
+            m_resultCollectionView.Scrolled -= OnCollectionViewScrolled;
```
Comment (Vetle444) [inline src/library/DIPS.Mobile.UI/Components/Searching/SearchPage.cs:104]: Vess man trøkk på et element, blir tastaturet lagt ned då, sia du fjerna unfocus?

Comment (Vetle444) [reply src/library/DIPS.Mobile.UI/Components/Searching/SearchPage.cs:104]: Altså vess man naviger

Comment (haavamoa) [reply src/library/DIPS.Mobile.UI/Components/Searching/SearchPage.cs:104]: Ah selvsagt, ska test :) Litt rart at å ikke trykk på et element gir fokus et annet sted imo.

Comment (Vetle444) [reply src/library/DIPS.Mobile.UI/Components/Searching/SearchPage.cs:104]: Heilt enig

**Review by Vetle444 — APPROVED** [review]

**Pull request merged**